tests/View/Mix/MixTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Illuminate\Tests\View\Mix;
+
+use PHPUnit\Framework\TestCase;
+use Illuminate\Support\HtmlString;
+
+class MixTest extends TestCase
+{
+    public function setUp()
+    {
+        app()->singleton('path.public', function () {
+            return __DIR__;
+        });
+    }
+
+    private function getMix()
+    {
+        return new \Illuminate\View\Mix\Mix;
```

Comment:
Yeah, but you can understand that this is much better than before, and you agree that each commit I'm making, you add some things.
I'm ok to discuss, exchange some "good" practices, and I agree than I have not your experience about the coding style of this framework. It's why I used your advices and pieces of code, even if sometimes I don't agree with. 

But please if we can just tell everything once, and don't come back each time on the same piece of code, we will save time both. Especially like here when it's here in a test, than even didn't exist!! 

**Imho you will have a lot more value to write tests than just make PRs to rewrite docblocks  and code style of others!**

I personally have a job, make opensource on my free time because I love it, and you are just disgusting me about that and it's a bit too bad...

If you want, you can make a PR to my branch I will merge it immediately, and so, it's ok we're done and we can move to important stuffs!